integration-libs/epd-visualization/components/visual-viewer/visual-viewer.service.ts

```diff
@@ -987,47 +985,55 @@ export class VisualViewerService implements OnDestroy {
       .epdVisualization as EpdVisualizationInnerConfig;
     const ui5Config = epdVisualization.ui5 as Ui5Config;
 
-    return new Observable((subscriber) => {
-      if (this.isUi5BootStrapped()) {
-        subscriber.next();
-        subscriber.complete();
-        return;
-      }
+    return new Observable(
+      (subscriber: {
+        next: () => void;
+        complete: () => void;
+        error: (error: any) => void;
```

Comment:
[Nitpick] Why do we need to add those types here? Normally the subscriber's type is derived from the definition of `Observable`'s constructor's argument. See for example: https://github.com/SAP/spartacus/blob/2a615c783407cc476c1097227f03fee9a75e38bc/projects/core/src/i18n/i18next/i18next-translation.service.ts#L36

That said, because of time constraints, we can turn a blind eye to this issue.

Reply:
VSCode was giving a warning about an implicit any type, so I applied the suggested fix. I can remove the inline type definition.